src/data/full-quiz.js

```diff
@@ -1,4 +1,59 @@
 const fullQuiz = [
+  {
+    Question:
+      "Which method returns an HTMLCollection object that consists of all of the elements that have the same class name?",
+    Answer: "getElementsByClassName()",
+    Distractor1: "querySelector()",
+    Distractor2: "getElementsByName()",
+    Distractor3: "getElementsByTagName()",
+    Explanation:
+    "The getElementsByClassName method of Document interface returns an array-like object of all child elements which have all of the given class name(s).",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementsByClassName",
+  },
+{
+    Question:
+      "Which operator is used to concatenate strings in JavaScript?",
+    Answer: "+",
+    Distractor1: "*",
+    Distractor2: "&",
+    Distractor3: "#",
+    Explanation:
+    "The addition operator (+) produces the sum of numeric operands or string concatenation.",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Addition",
+  },
+{
+    Question:
+      "What method is used to arrange array values into alphabetical and/or ascending order?",
+    Answer: "sort()",
+    Distractor1: "shift()",
+    Distractor2: "unshift()",
+    Distractor3: "from()",
+    Explanation:
+    "The sort() method sorts the elements of an array in place and returns the sorted array. The default sort order is ascending, built upon converting the elements into strings, then comparing their sequences of UTF-16 code units values",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort",
+  },
+{
+    Question:
+      "What method is used to arrange array values into descending order?",
+    Answer: "reverse()",
+    Distractor1: "from()",
+    Distractor2: "unshift()",
+    Distractor3: "sort()",

```

Comment:
I think we should change the answer for Distractor3 to be one of the more obviously wrong answers.
That way reverse() is the only logical choice especially for those newer to programming that might choose sort. 
We try to make these questions on the easier side so it is accessible to all levels. 
There will be future versions of the site, where people can choose quiz difficulty levels.  But for now, easier is better.

Proposed fix: join()

Reply:
Sounds good. Thank you for the feedback. :-)